addons/l10n_br_website_sale/static/src/js/address.js

```diff
@@ -0,0 +1,76 @@
+/** @odoo-module **/
+
+import websiteSaleAddress from '@website_sale/js/address';
+
+websiteSaleAddress.include({
+    events: Object.assign(
+        {},
+        websiteSaleAddress.prototype.events,
+        {
+            'input input[name="zip"]': '_onChangeZip',
```

Comment:
As mentioned, setting the zipcode then the country doesn't update trigger _onChangeZip either, it should imo as the fields are put in that order (with a filtering not doing anything if it's not the right country ofc)

Reply:
Sorry @JulienAlardot, I missed that in your initial review. That makes sense. It's a one-line change, there's already a `_changeCountry` override that filters on Brazil so I call `_onChangeZip` there now. I also test this in the tour.